enterprise/server/test/integration/podman/podman_test.go

```diff
@@ -13,12 +13,15 @@ import (
 
 	"github.com/buildbuddy-io/buildbuddy/enterprise/server/remote_execution/commandutil"
 	"github.com/buildbuddy-io/buildbuddy/enterprise/server/remote_execution/container"
+	_ "github.com/buildbuddy-io/buildbuddy/enterprise/server/remote_execution/containers/docker"
```

Comment:
nit: can you move this to the bottom of the import section and add a comment that this is included for flags?

Reply:
Oh shoot, didn't realize those were supposed to be at the bottom as well. Fixed.